samples/web/content/apprtc/js/main.js

```diff
@@ -345,16 +367,62 @@ function displayError(error) {
   showInfoDiv();
 }
 
-function toggleFullScreen() {
-  try {
-    // TODO: add shim so not Chrome only
-    if (document.webkitIsFullScreen) {
-      document.webkitCancelFullScreen();
-    } else {
-      videosDiv.webkitRequestFullScreen();
-      remoteCanvas.webkitRequestFullScreen();
-    }
-  } catch (event) {
-    trace(event);
+function toggleFullscreen(){
+  if (isFullScreen()) {
+    document.cancelFullScreen();
+  } else {
+    document.body.requestFullScreen();
   }
+  fullscreenIconSet.toggle();
+}
+
+function $(selector){
+  return document.querySelector(selector);
+}
+
+function hide(element){
+  element.classList.add('hidden');
 }
+
+function show(element){
+  element.classList.remove('hidden');
+}
+
+function activate(element){
+  element.classList.add('active');
+}
+
+function deactivate(element){
+  element.classList.remove('active');
+}
+
+function IconSet(icon0, icon1){
+  this.icon0 = document.querySelector(icon0);
+  this.icon1 = document.querySelector(icon1);
+}
+
+IconSet.prototype.toggle = function() {
+  if (this.icon0.classList.contains('hidden')){
```

Comment:
suggest adding a toggle() function so that IconSet.prototype.toggle just becomes { toggle(icon0); toggle(icon1); }